### ISSUE TITLE
After a second file change, Webpack-encore builds the files but does not write the changes into folder.

### ISSUE THREAD

**Issue by getJv** [symfony/webpack-encore#1209]
Hello all, Im facing a weird behaviour in my Symfony 6 with Encore and React 18.

I set a fresh new symfony 6 project, installed the Encore pack and enabled the ReactPreset. 

When I ran 'npm run watch' it Compliles and set React working perfectly well. on my browser. 

- if I change the JSX files the watch mode says it was compiled and XX files was written into public folder. it the content into the browser is updated (first changes works well as suppose to be...)
- BUT after a second change it compile, say it was written BUT the public folder is no update with the new compiled changes... 

I added all files here in this issue, if is needed to copy and past here please let me know

https://stackoverflow.com/questions/76252739/symfony-encore-webpack-builds-react-changes-but-does-not-updates-the-output-publ

Thank you so much for your time. I already tried several different configuration and read a lot about but now i got out of ideas...

**Comment by weaverryan** [comment]
Hmm, that is strange behavior. I really have no idea - we're not doing anything particularly special. If Webpack says it's outputting files, it usually is! You could try `.enableVersioning()` (with no arguments) so that you always get versioned filenames - maybe that will help.

Good luck - sorry I can't help more!

**Comment by getJv** [comment]
Hey @weaverryan thank you so much for your time. Im here to tell I've found the solution. 

it is so simple the its a shame... I already set up several projects but this was really tough because there wasn't any error message to debug it.

Basically it  was a file not named in the capital casa BUT imported as it was.

Im adding this here to help someone else. maybe even someone to add this error message to webpack/encore...

![image](https://github.com/symfony/webpack-encore/assets/2864961/3b740cb3-6b00-4134-a79a-6071980edd79)

After renaming the file from app.js to App.js was normally solved.

Good coding guys.
